kitsune/messages/utils.py

```diff
@@ -18,22 +19,37 @@ def send_message(to, to_group=None, text=None, sender=None):
 
     # Assuming OutboxMessage doesn't necessarily need to be created for every recipient
     if to:
+        # This is users from the To field
+        users = {user for user in to if isinstance(user, User)}
+        # This is groups from the To field
+        groups = [group for group in to if isinstance(group, Group)]
```

Comment:
Is there a reason why `groups` should not be a set like `users`?

Reply:
No, no reason at all.